src/git_changelog/cli.py

```diff
@@ -353,12 +443,6 @@ def build_and_render(
     # get provider
     provider_class = providers[provider] if provider else None
 
-    # TODO: remove at some point
-    if bump_latest:
-        warnings.warn("`bump_latest=True` is deprecated in favor of `bump='auto'`", DeprecationWarning, stacklevel=1)
-        if bump is None:
-            bump = "auto"
-
```

Comment:
This should stay as users can call `build_and_render` from Python directly.

Reply:
Removed. I've also removed the other warning, as it doesn't make a lot of sense to shout at the user twice.

Reply:
Reinstated, and added a temporary test to ensure the warnings are issued when they should.